crates/rattler_solve/tests/snapshots/backends__resolvo__solve_dummy_repo_optional_depends_feature_conflict.snap

```diff
@@ -0,0 +1,11 @@
+---
+source: crates/rattler_solve/tests/backends.rs
+expression: result.unwrap_err()
+---
+Cannot solve the request because of: The following packages are incompatible
+├─ foo >=4.0 can be installed with any of the following options:
+│  └─ foo 4.0.2
+└─ foo[with-bar] * cannot be installed because there are no viable options:
+   └─ foo[with-bar] 3.0.2 would require
+      └─ foo ==3.0.2 py36h1af98f8_1[md5=fb731d9290f0bcbf3a054665f33ec94f, sha256=67a63bec3fd3205170eaad532d487595b8aaceb9814d13c6858d7bac3ef24cd4], which cannot be installed because there are no viable options:
```

Comment:
Mm this error is pretty uninformative. Especially if there will be multiple solvables that provide that extra. It will probably list every single solvable and extra here. 

Perhaps it would be possible to merge all the SolvableWithFeature solvables of a single version into a single solvable? I assume that will result in a much clearer error message and reduce the total number of solvables..